dlt/common/libs/pydantic.py

```diff
@@ -0,0 +1,48 @@
+from typing import Type, Union, get_type_hints, get_args
+
+from dlt.common.exceptions import MissingDependencyException
+from dlt.common.schema.typing import TTableSchemaColumns
+from dlt.common.data_types import py_type_to_sc_type, TDataType
+from dlt.common.typing import is_optional_type, extract_inner_type, is_list_generic_type, is_dict_generic_type, is_union
+
+try:
+    from pydantic import BaseModel, Field
+except ImportError:
+    raise MissingDependencyException("DLT pydantic Helpers", ["pydantic"], "DLT Helpers for for pydantic.")
+
+
+def pydantic_to_table_schema_columns(model: Union[BaseModel, Type[BaseModel]]) -> TTableSchemaColumns:
```

Comment:
this is good. nullability is set and code to infer type reused :)

could we add one optional parameter: `skip_complex_types` default to False. if set we do not add columns with complex types at all to generated schema. that would mimic the default dlt behaviour that creates child tables.